apps/aehttp/src/sc_ws_handler.erl

```diff
@@ -329,6 +329,38 @@ process_incoming(#{<<"action">> := <<"get">>,
         {error, _} ->
             {reply, error_response(R, invalid_arguments)}
     end;
+process_incoming(#{<<"action">> := <<"get">>,
+                   <<"tag">> := <<"poi">>,
+                   <<"payload">> := #{<<"accounts">>  := AccountsE,
+                                      <<"contracts">>  := ContractsE}} = R, State) ->
+    Parse =
+        fun(T, Keys) ->
+            try {ok, lists:foldr(
+                      fun(K, Acc) ->
+                              {ok, Res} = aec_base58c:safe_decode(T, K),
+                              [Res | Acc]
+                      end, [], Keys)}
+            catch
+                error:_ ->
+                    {error, invalid_pubkey}
+            end
+        end,
+    case {Parse(account_pubkey, AccountsE), Parse(contract_pubkey, ContractsE)} of
+        {{ok, Accounts0}, {ok, Contracts0}} ->
+            Accounts = [{account, A} || A <- Accounts0 ++ Contracts0],
+            Contracts = [{contract, C} || C <- Contracts0],
+            case aesc_fsm:get_poi(fsm_pid(State), Accounts ++ Contracts) of
+                {ok, PoI} ->
+                    Resp = #{action => <<"get">>,
+                             tag => <<"poi">>,
+                             payload => #{
+                                <<"poi">> => aec_base58c:encode(poi, aec_trees:serialize_poi(PoI))}},
+                    {reply, Resp};
+                {error, Reason} ->
+                    {reply, error_response(R, Reason)}
+            end;
+        _ -> {reply, error_response(R, broken_hexcode)}
```

Comment:
I suspect user would prefer a more specific error - i.e. on which of the two values the error is. Though seems good enough.